src/components/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import {Link} from 'react-router-dom'
+
+const Header = () => {
+return(
+<header id='header'>
+     <h1 className='heading'>NC NewsIt</h1>
+     <nav>
+          <Link to='/'><button className='headerButton'>Articles</button></Link>
+          <Link to='/profile'><button className='headerButton'>Profile</button></Link>
```

Comment:
Button elements shouldn't be for navigation, better to just have links rather than both